### PR TITLE
Additional GNSS field added

### PR DESCRIPTION
**Pull request by sumps**
Quality, Integrity and PDOP  added.

### CODE REVIEW

File: schemas/groups/navigation.json [hunk 1]
```diff
@@ -169,9 +169,38 @@
...
+            "Estimated (DR) mode",
+            "Manual input",
+            "Simulator mode",
+            "Error"
+          ]
+        },
+        
+        "integrity": {
+          "type": "object",
+          "description": "Integrity of the satellite fix"
```
Comment (tkurki) [inline schemas/groups/navigation.json]: Missing comma

File: schemas/groups/navigation.json [hunk 2]
```diff
@@ -187,7 +216,12 @@
 
         "horizontalDilution": {
           "$ref": "../definitions.json#/definitions/numberValue",
-          "description": "Horizontal position error"
+          "description": "Horizontal Dilution of Precision"
+        },
+
+        "positionlDilution": {
```
Comment (tkurki) [inline schemas/groups/navigation.json]: Typo

File: schemas/groups/navigation.json [hunk 3]
```diff
@@ -169,9 +169,38 @@
...
         },
 
-        "quality": {
-          "$ref": "../definitions.json#/definitions/numberValue",
-          "description": "Quality of the satellite fix"
+        "methodQuality": {
+          "type": "object",
+          "description": "Quality of the satellite fix",
+          "properties": {
+        "value": {
```
Comment (tkurki) [inline schemas/groups/navigation.json]: Indentation

File: schemas/groups/navigation.json [hunk 4]
```diff
@@ -169,9 +169,38 @@
...
+            "Precise GNSS",
+            "RTK fixed integer",
+            "RTK float",
+            "Estimated (DR) mode",
+            "Manual input",
+            "Simulator mode",
+            "Error"
+          ]
+        },
+
```
Comment (tkurki) [inline schemas/groups/navigation.json]: Unbalanced curly braces

File: schemas/groups/navigation.json (hunk 3 above)
Comment (timmathews) [inline schemas/groups/navigation.json]: Can you explain `methodQuality`? The name doesn't really make sense to me

File: schemas/groups/navigation.json [hunk 5]
```diff
@@ -169,9 +169,40 @@
...
+                "DGNSS fix",
+                "Precise GNSS",
+                "RTK fixed integer",
+                "RTK float",
+                "Estimated (DR) mode",
+                "Manual input",
+                "Simulator mode",
+                "Error"
+            ]
+          }
```
Comment (tkurki) [inline schemas/groups/navigation.json]: MIssing closing curly brace

File: schemas/groups/navigation.json [hunk 6]
```diff
@@ -169,9 +169,40 @@
...
+          "description": "Integrity of the satellite fix",
+          "properties": {
+            "value": {
+            "type": "string",
+            "enum": [
+              "no Integrity checking",
+              "Safe",
+              "Caution",
+              "Unsafe"
+            ]
```
Comment (tkurki) [inline schemas/groups/navigation.json]: MIssing closing curly brace

**Pull request merged**